hbase-server/src/main/java/org/apache/hadoop/hbase/master/normalizer/SimpleRegionNormalizer.java

```diff
@@ -276,18 +273,22 @@ private double getAverageRegionSizeMb(final List<RegionInfo> tableRegions) {
       LOG.warn("TableDescriptor for {} unavailable, table-level target region count and size"
         + " configurations cannot be considered.", table, e);
     }
-
-    double avgRegionSize;
     if (targetRegionSize > 0) {
       avgRegionSize = targetRegionSize;
-    } else if (targetRegionCount > 0) {
-      avgRegionSize = totalSizeMb / (double) targetRegionCount;
     } else {
-      avgRegionSize = totalSizeMb / (double) regionCount;
+      final int regionCount = tableRegions.size();
+      final long totalSizeMb = tableRegions.stream()
+        .mapToLong(this::getRegionSizeMB)
+        .sum();
```

Comment:
Sorry, just to be sure on the goal here, this is just to avoid this loop always happen, even when totalSizeMb is not been used?

Reply:
Yes, this is origin goal. If we set  NormalizerTargetRegionSize, we use NormalizerTargetRegionSize as avgRegionSize and return. But now I found another serious bug, NormalizerTargetRegionCount and NormalizerTargetRegionSize will never used when log level is not debug.